### PR TITLE
Update sdk

### LINKED ISSUE #170: 🐞 [Bug]: Do not give error for `wipe` if `summary` is not found
### What happened?

When `wipe` command tries to delete everything, and if `summary` is missing, it crashes with an error message. 

### Version

0.3.0

### platform

- [X] linux
- [X] windows
- [X] macos

### Expected behavior

Instead of crash, it could simply print this out

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ozgunozerk**
This ps:
- gets rid of `unstable_tokio` for default use case
- separates tests into its own file (since util.rs started to get fat)
- updates sdk revision
- refactoring `summary` structs 
- closes #170

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
     runs-on: ${{ matrix.build.os }}
 
     env:
-      RUSTFLAGS: ${{ matrix.build.rustflags }} --cfg tokio_unstable
```
Comment (i1i1) [inline .github/workflows/release.yml:59]: I would better compile release with debug. WDUT?

File: src/tests.rs
Comment (i1i1) [inline src/tests.rs:1]: I would better move file into `tests/` folder

Comment (ozgunozerk) [reply src/tests.rs:1]: that was my first attempt. Then I thought, these tests are not integration tests, but unit tests. So I followed monorepo's farmer library approach.

Comment (i1i1) [reply src/tests.rs:1]: Yeah, makes sense then

File: src/commands/farm.rs [hunk 2]
```diff
@@ -67,7 +67,7 @@ pub(crate) async fn farm(is_verbose: bool, executor: bool) -> Result<()> {
         }
     }
 
-    let summary = Summary::new(Some(farmer_config.plot_size)).await?;
+    let summary = Summary::new(Some(farmer_config.clone().plot_size)).await?;
```
Comment (i1i1) [inline src/commands/farm.rs]: Do you need clone here though :thinking: ?

Comment (ozgunozerk) [reply src/commands/farm.rs]: yes, `copy` is not implemented for `bytesize` wrapper on sdk. No big deal, we can fix it later :)

Comment (i1i1) [reply src/commands/farm.rs]: Okay, can you move clone to `plot_size` then? Like `farmer_config.plot_size.clone()`?

Comment (ozgunozerk) [reply src/commands/farm.rs]: lemme try sure

**Review by i1i1 — DISMISSED** [review]

**Review by i1i1 — APPROVED** [review]

**Pull request merged**